Bitbucket-Pipelines/two-stage-pipeline-template/{{cookiecutter.outputDir}}/bitbucket-pipelines.yml

```diff
@@ -0,0 +1,177 @@
+              {%- if cookiecutter.prod_image_repository %}
+              --image-repository ${TESTING_IMAGE_REPOSITORY}
+              {%- endif %}
+              --no-fail-on-empty-changeset
+              --role-arn ${TESTING_CLOUDFORMATION_EXECUTION_ROLE}
+          services:
+            - docker
+
+    {{ cookiecutter.main_git_branch }}:
+      - step:
+          name: Build and Package
+          script:
+            - export SAM_TEMPLATE="{{ cookiecutter.sam_template }}"
+            - export PIPELINE_USER_ACCESS_KEY_ID=${{ "{" }}{{cookiecutter.pipeline_user_aws_access_key_id_variable_name}}{{ "}" }}
+            - export PIPELINE_USER_SECRET_ACCESS_KEY=${{ "{" }}{{cookiecutter.pipeline_user_aws_secret_access_key_variable_name}}{{ "}" }}
+            - export TESTING_PIPELINE_EXECUTION_ROLE="{{ cookiecutter.testing_pipeline_execution_role }}"
+            - export TESTING_CLOUDFORMATION_EXECUTION_ROLE="{{ cookiecutter.testing_cloudformation_execution_role }}"
+            - export TESTING_ARTIFACTS_BUCKET="{{ cookiecutter.testing_artifacts_bucket }}"
+            - export TESTING_REGION="{{ cookiecutter.testing_region }}"
+            {%- if cookiecutter.testing_image_repository %}
+            - export TESTING_IMAGE_REPOSITORY="{{ cookiecutter.testing_image_repository }}"
+            {% else %}
+            # If there are functions with "Image" PackageType in your template,
+            # uncomment the line below and add "--image-repository ${TESTING_IMAGE_REPOSITORY}" to
+            # testing "sam package" and "sam deploy" commands.'
+            #- export TESTING_IMAGE_REPOSITORY='0123456789.dkr.ecr.region.amazonaws.com/repository-name'
+            {% endif -%}
+            - export PRODUCTION_PIPELINE_EXECUTION_ROLE="{{ cookiecutter.prod_pipeline_execution_role }}"
+            - export PRODUCTION_CLOUDFORMATION_EXECUTION_ROLE="{{ cookiecutter.prod_cloudformation_execution_role }}"
+            - export PRODUCTION_ARTIFACTS_BUCKET="{{ cookiecutter.prod_artifacts_bucket }}"
+            - export PRODUCTION_REGION="{{ cookiecutter.prod_region }}"
+            {%- if cookiecutter.prod_image_repository %}
+            - export PRODUCTION_IMAGE_REPOSITORY="{{ cookiecutter.prod_image_repository }}"
+            {% else %}
+            # If there are functions with "Image" PackageType in your template,
+            # uncomment the line below and add "--image-repository ${PRODUCTION_IMAGE_REPOSITORY}" to
+            # testing "sam package" and "sam deploy" commands.'
+            #- export PRODUCTION_IMAGE_REPOSITORY='0123456789.dkr.ecr.region.amazonaws.com/repository-name'
+            {% endif -%}
+            - sam build --template $SAM_TEMPLATE --use-container
+            - source assume-role.sh $TESTING_PIPELINE_EXECUTION_ROLE testing-stage-packaging testing-stage
+            - >
+              sam package --profile testing-stage
+              --s3-bucket $TESTING_ARTIFACTS_BUCKET
```

Comment:
Can we add some identation on lines that are part of a command? Currently the whole template reads as a whole block, difficult to quickly scan where commands start.

Reply:
Sadly Nope :( adding indentation makes it ignore all the lines.